cpp/src/community/nvgraph_gdf.cu

```diff
@@ -259,11 +255,10 @@ void analyzeClustering_ratio_cut_nvgraph(Graph* gdf_G,
                                                   gdf_column* clustering,
                                                   float* score) {
 
-  CUGRAPH_EXPECTS(gdf_G != nullptr, "Invalid API parameter");
-  CUGRAPH_EXPECTS(gdf_G->adjList != nullptr, "Invalid API parameter");
-  CUGRAPH_EXPECTS(gdf_G->adjList->edge_data != nullptr, "Invalid API parameter");
-  CUGRAPH_EXPECTS(clustering != nullptr, "Invalid API parameter");
-  CUGRAPH_EXPECTS(clustering->data != nullptr, "Invalid API parameter");
+  CHECK_GRAPH(gdf_G);
+  CUGRAPH_EXPECTS(gdf_G->adjList->edge_data != nullptr, "Invalid API parameter: graph edge data NULL");
```

Comment:
Needs to be consistent with
https://github.com/rapidsai/cugraph/pull/720/files#diff-85ea04eb60e5e96b07932333ed67226cR171

CUGRAPH_EXPECTS(gdf_G->adjList->edge_data != nullptr, "Invalid API parameter: edge data is NULL");

Reply:
fixed